charts/camunda-platform/templates/NOTES.txt

```diff
@@ -142,3 +142,58 @@ Default user and password: "{{ .Values.identity.firstUser.username -}}/{{ .Value
 {{ end }}
 
 {{- end }}
+
+{{- if or (.Values.global.ingress.enabled) (.Values.identity.ingress.enabled) (.Values.operate.ingress.enabled) (.Values.optimize.ingress.enabled) (.Values.tasklist.ingress.enabled) ( index .Values "zeebe-gateway" "ingress" "enabled" ) (.Values.webModeler.ingress.enabled) }}
+
+### Ingress
+
+{{- if .Values.global.ingress.enabled }}
+
+#### Combined Ingress
+
+- Ingress:
+  - Enabled: {{ .Values.global.ingress.enabled }}
+  - Web Applications:
+    - Identity: https://{{ .Values.global.ingress.host }}{{ .Values.identity.contextPath }}
+    - Operate: https://{{ .Values.global.ingress.host }}{{ .Values.operate.contextPath }}
+    - Optimize: https://{{ .Values.global.ingress.host }}{{ .Values.optimize.contextPath }}
+    - Tasklist: https://{{ .Values.global.ingress.host }}{{ .Values.tasklist.contextPath }}
+    - WebModeler: https://{{ .Values.global.ingress.host }}{{ .Values.webModeler.contextPath }}
+  - Keycloak Authentication: https://{{ .Values.global.ingress.host }}/auth
```

Comment:
The context `/auth` is configurable and could be different.
Use `{{ include "identity.keycloak.contextPath" .Subcharts.identity }}` [as used in the Ingress](https://github.com/camunda/camunda-platform-helm/blob/6ec2184f3d76de6b7eea6e20cbe383b8109f26f1/charts/camunda-platform/templates/ingress.yaml#LL28C19-L28C84).